### ISSUE TITLE
Add importable VRC Session Profiles

### ISSUE THREAD

**Issue by erikquinn** [vZMA/ZMA#105]
To prevent wasted time, let's generate the profiles how we want them for all controlled airports, and then people can just load them in, rather than needing to generate them all on their own.

**Comment by FriedrichKayak** [comment]
I am almost certain that if I set up these profiles, people will complain about how I did it. Does someone have a profile that is a candidate to become the standard?

**Comment by FriedrichKayak** [comment]
Closed as duplicate of #102

**Comment by FriedrichKayak** [comment]
Oops, wanted to leave this one open.

**Comment by FriedrichKayak** [comment]
created a /profiles directory in this repo for these items; will start with a "base" from which the individual positions are derived

**Comment by migte** [comment]
Sample of result: 52ea988

**Comment by migte** [comment]
Disregard the above, I have made a few demo profiles for ground. Need this to be reviewed prior to determining this is the format we are looking for. https://github.com/vZMA/ZMA/tree/%23105-VRC-PRF/docs/profiles

**Comment by migte** [comment]
Looking into the possibility of a ZMA installer. You would run the exe and it downloads all the sector files and everything into a folder. Whenever you restart PC it checks for updates and runs the updater if there is an update for the sector files. This also allows for the implementation of multiple POF and sector files. Kind of weird but I'll make a demo to explain my idea.

**Comment by migte** [comment]
yeah disregard that idea above for now. It is completely possible, in fact I already made it, but I doubt many people would use it in the first place. maybe someday in the distant future.

**Comment by erikquinn** [comment]
Many facilities use a dropbox to be able to push updated sector files and such without requiring the controllers to retrieve it. vSTARS/vERAM would still need to be re-imported, but VRC would just pull the modified file when it loads, and voila. I don't know what the best approach would be, or if it were something worth pursuing, but just throwing out options about a possible implementation.

**Comment by migte** [comment]
That would be a good solution. Not sure how dropbox works, but if we are able to place the drobpox files outside of the Users section, it could allow for a very plug and play experience. My primary concern is really expecting the controllers to select the proper POF and alias file for every facility they open. Would be much easier if the file path was already predefined for them.

**Comment by erikquinn** [comment]
Would also allow VRC users to experience the same benefit of the proposed vSTARS handoffs without the facility character (ie `D` to hand off to south feeder rather then the more verbose `MD`), without any added inconvenience...

**Comment by migte** [comment]
With the announcement of the termination of future support for VRC, I wonder if setting up an infrastructure as such is even worth it. Perhaps just best to provide files and kindly ask users to place them in a specific location for them to function properly.
